layout/_partials/footer.swig

```diff
@@ -1,6 +1,9 @@
 <div class="copyright">{#
 #}{% set current = date(Date.now(), "YYYY") %}{#
-#}&copy; {% if theme.footer.since and theme.footer.since != current %}{{ theme.footer.since }} – {% endif %}{#
+#}{% if theme.footer.beian.enable %}{#
+#}<a target="_blank" rel="external nofollow" href="http://www.miitbeian.gov.cn/"> {{theme.footer.beian.icp}} </a>{#
```

Comment:
I opened the "beian" option today and found that the "beian" option and "since2015" are crowded in one line. It is not very beautiful. Can I make the "beian" information text appear on a separate line?

Reply:
You can create your own pull request with this changes. Welcome!

Reply:
> You can create your own pull request with this changes. Welcomeo'k!
ok，thanks reply。